libs/shared/ui/src/libs/theme/tokens/spacing.ts

```diff
@@ -0,0 +1,17 @@
+declare module '@mui/material/styles' {
+  interface Theme {
+    space: {
+      [key: string]: number
+    }
+  }
+  interface ThemeOptions {
+    space: {
```

Comment:
This should be optional
```suggestion
    space?: {
```